modules/sdk-coin-avaxp/src/avaxp.ts

```diff
@@ -78,16 +137,16 @@ export class AvaxP extends BaseCoin {
     switch (explainedTx.type) {
       case TransactionType.AddDelegator:
       case TransactionType.AddValidator:
-        if (!params.txParams.recipients || params.txParams.recipients.length === 0) {
-          const filteredRecipients = [{ address: stakingOptions.nodeID, amount: stakingOptions.amount }];
-          const filteredOutputs = explainedTx.outputs.map((output) => _.pick(output, ['address', 'amount']));
-
-          if (!_.isEqual(filteredOutputs, filteredRecipients)) {
-            throw new Error('Tx outputs does not match with expected txParams');
-          }
-          if (stakingOptions.amount !== explainedTx.outputAmount) {
-            throw new Error('Tx total amount does not match with expected total amount field');
-          }
+        if (params.txParams.recipients && params.txParams.recipients.length !== 0) {
+          throw new Error('Stake Tx does not require recipients');
+        }
+        this.validateStakingTx(stakingOptions, explainedTx);
+        break;
+      case TransactionType.Export:
+        if (!params.txParams.recipients) {
+          throw new Error('Export Tx requires a recipient');
+        } else {
+          this.validateExportTx(params.txParams.recipients, memo?.value, explainedTx);
```

Comment:
Please, check memo exist, like you do with recipients. 

We have a technical debt 
https://bitgoinc.atlassian.net/browse/BG-56877